aten/src/ATen/native/quantized/affine_quantizer.cpp

```diff
@@ -57,7 +59,7 @@ void checkQuantizedTensor(const std::string& fn_name, Tensor t) {
 }
 
 template <typename T>
-void checkZeroPoint(const std::string& fn_name, int64_t zero_point) {
+void checkZeroPoint(const std::string& fn_name, T zero_point) {
```

Comment:
I think `T` here is for things like `int8`, `uint8` etc.?

Reply:
Yeah we don't need to check zero_point when it is float.